### PR TITLE
Becs

### PR DESCRIPTION
**Pull request by rebeccajackson**

### CODE REVIEW

File: public/main.css [hunk 1]
```diff
@@ -170,175 +170,94 @@ body {
...
 }
 
 .topRecentTi3 {
-  grid-column: 1 / 2;
-  grid-row: 8 / 10;
-  max-height: 145px;
+  margin: 1em;
 }
 
 .topRecentTi4 {
```
Comment (GBarrBrown) [inline public/main.css]:
```suggestion

```

File: public/main.css [hunk 2]
```diff
@@ -170,175 +170,94 @@ body {
...
-  grid-column: 1 / 2;
-  grid-row: 10 / 12;
-  max-height: 145px;
-}
-
-.topContribs {
-  grid-column: 8 / 12;
-  grid-row: 7 / 11;
-  background-color: white;
+  margin: 1em;
```
Comment (GBarrBrown) [inline public/main.css]:
```suggestion
```

File: public/main.css [hunk 3]
```diff
@@ -170,175 +170,94 @@ body {
...
 }
```
Comment (GBarrBrown) [inline public/main.css]:
```suggestion
```

File: public/main.css [hunk 4]
```diff
@@ -170,175 +170,94 @@ body {
...
 
 .topRecentTi2 {
-  grid-column: 1 / 2;
-  grid-row: 6 / 8;
-  max-height: 145px;
+  margin: 1em;
 }
 
 .topRecentTi3 {
```
Comment (GBarrBrown) [inline public/main.css]:
```suggestion
```

File: public/main.css [hunk 5]
```diff
@@ -170,175 +170,94 @@ body {
...
-  grid-column: 1 / 2;
-  grid-row: 8 / 10;
-  max-height: 145px;
+  margin: 1em;
```
Comment (GBarrBrown) [inline public/main.css]:
```suggestion
```

**Review by GBarrBrown — DISMISSED** [review]
Please remove unnecessary .topRecentTi(n)'s from .css
Everything else looks good thought, fix and make a new PR

**Comment by rebeccajackson** [comment]
I will test that css and remove if unecessary

**Pull request merged**